src/plugins/plugin-koa.ts

```diff
@@ -86,7 +86,7 @@ function startSpanForRequest<T>(
     // middlewear runs.
     root.addLabel(api.labels.HTTP_METHOD_LABEL_KEY, req.method);
     root.addLabel(api.labels.HTTP_URL_LABEL_KEY, url);
-    root.addLabel(api.labels.HTTP_SOURCE_IP, req.connection.remoteAddress);
+    root.addLabel(api.labels.HTTP_SOURCE_IP, req.ip);
```

Comment:
It seems like `req` (= `ctx.req`) here is a raw `IncomingMessage`, so TS doesn't recognize `req.ip` as a field. Maybe this is what we want instead?

```suggestion
    root.addLabel(api.labels.HTTP_SOURCE_IP, ctx.request.ip);
```

Reply:
Yup, sorry I've made the changes without vscode 🤐

Reply:
No problem! Thanks for the contribution!

Reply:
Cheers!